geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -41,6 +41,8 @@ start server \
   --redis-password=<redisPassword>
 ```
 
+Note: if any of redis-bind-address, redis-password, or redis-port are included, a compatible-with-Redis geode server will be started
```

Comment:
Please revise this sentence with some formatting and capitalization changes. I suggest:

If any of the options `redis-bind-address`, `redis-password`, or `redis-port` are included, a compatible-with-Redis Geode server will be started.

Reply:
changed.  Thanks